### PR TITLE
Add enhanced search functionality to babes.com

### PR DESCRIPTION
**Pull request by aicha1566**
Leaving sceneID as the first searchable method because the babes.com search engine seems fairly inaccurate. It works better for recent releases however.

### CODE REVIEW

File: Contents/Code/siteBabes.py [hunk 1]
```diff
@@ -5,25 +5,42 @@
...
+    # Use encodedTitle as sceneId, if provided
     sceneID = encodedTitle.split('%20', 1)[0]
     Log("SceneID: " + sceneID)
     try:
         sceneTitle = encodedTitle.split('%20', 1)[1].replace('%20',' ')
     except:
         sceneTitle = ''
     Log("Scene Title: " + sceneTitle)
-    url = PAsearchSites.getSearchSearchURL(siteNum) + sceneID + "/1"
+    url = "https://www.babes.com/scene/" + sceneID + "/1"
```
Comment (SgtBatten) [inline Contents/Code/siteBabes.py]:
```suggestion
    url = "PAsearchSites.getSearchBaseURL(siteNum) + "/scene/" + sceneID + "/1"
```

Comment (SgtBatten) [inline Contents/Code/siteBabes.py]:
```suggestion
    url = "PAsearchSites.getSearchBaseURL(siteNum) + "/scene/" + sceneID + "/1"
```

**Comment by aicha1566** [comment]
Applied the suggested change

**Comment by aicha1566** [comment]
Thanks for the suggestions

**Pull request merged**